docs/reference/mapping/params/index-prefixes.asciidoc

```diff
@@ -0,0 +1,64 @@
+[[index-prefixes]]
```

Comment:
Can you successfully build the docs? I would have expected that you would need to add an `include` for this file to `docs/reference/mapping/params.asciidoc`?

Reply:
It's referred to from `docs/references/mapping/types/text.asciidoc` so the build passes.  I'll add it to `params.asciidoc` as well though.